### DISCUSSION TITLE
LaserGRBL on Linux Mint

### DISCUSSION THREAD

**Discussion by G-Jim** [arkypita/LaserGRBL#1763] [Q&A] [answered]
I'm attempting to use LaserGRBL with Linux Mint 20.3 to connect to Ortur Laser Master 2 Pro S2.

I can connect, with the same compute, using Win10, but I prefer to Linux.

LaserGRBL requests a COM port #, Linux Mint doesn't have those.  Anyone successfully using LaserGRBL on a Linux machine?

Thanks for any assistance,
Jim

**Comment by arkypita** [comment, marked as answer]
Did you read this thread? https://github.com/arkypita/LaserGRBL/issues/5#issuecomment-873564055

**Reply by G-Jim** [reply]
Thanks, I'm now able to control the Ortur under Linux Mint.
